source/simulator/parameters.cc

```diff
@@ -884,7 +884,7 @@ namespace aspect
                          Patterns::List(Patterns::Anything()),
                          "A user-defined name for each of the compositional fields requested.");
       prm.declare_entry ("Compositional field methods", "",
-                         Patterns::List (Patterns::Selection("field|particles|static")),
+                         Patterns::List (Patterns::Selection("field|particles|static|melt field")),
```

Comment:
is there a reason the enum is called differently?

Reply:
What do you mean? The reason why the enum has a different name than the input parameter? I have no idea, but I think AdvectionFieldMethod makes a lot more sense as a name, and we can not change the name of the input parameter without breaking backwards compatibility.

Reply:
no, I mean why is the parameter "melt field" while the enum is "fem melt field"

Reply:
Oh, I just copied the existing structure. For the normal compositional field, the parameter is "field", and the enum is "fem_field".